sdk/cosmos/azure-cosmos/azure/cosmos/aio/_container.py

```diff
@@ -815,11 +815,22 @@ async def delete_all_items_by_partition_key(
         partition_key: Union[str, int, float, bool],
         **kwargs: Any
     ) -> None:
-        """Exposes an API to delete all items with a single partition key without the user having
-                 to explicitly call delete on each record in the partition key.
+        """The delete by partition key feature is an asynchronous, background operation that allows you to delete all
+        documents with the same logical partition key value, using the Cosmos SDK. The delete by partition key
+        operation is constrained to consume at most 10% of the total
+        available RU/s on the container each second. This helps in limiting the resources used by
+        this background task.
 
         :param partition_key: Partition key for the items to be deleted.
         :type partition_key: Any
+        :keyword str pre_trigger_include: trigger id to be used as pre operation trigger.
+        :keyword str post_trigger_include: trigger id to be used as post operation trigger.
+        :keyword str session_token: Token for use with Session consistency.
+        :keyword dict[str,str] initial_headers: Initial headers to be sent as part of the request.
```

Comment:
let's remove initial_headers from the documentation for this method and for patch api actually, sorry I missed the fact I added it there as well - we should move towards not supporting this option for users and having them instead pass us the options to set the headers within the SDK